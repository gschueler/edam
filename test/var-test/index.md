% Test

Testing variables:

This should be abc:

    ${Test-1}

This should be 123:

    ${Test-2.test}
