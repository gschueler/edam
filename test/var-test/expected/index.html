<!DOCTYPE html>
<html>
<head>
  <title>Test</title>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta name="generator" content="pandoc" />
  <link rel="stylesheet" href="style.css" type="text/css" />
</head>
<body>


<nav class="page top">
    <ul>
        <li class="current"><a href="index.html">Test</a></li>
        
        
        
    </ul>
</nav>
<div id="docbody">
<h1 class="title">Test</h1>
<p
>Testing variables:</p
><p
>This should be abc:</p
><pre
><code
  >abc
</code
  ></pre
><p
>This should be 123:</p
><pre
><code
  >123
</code
  ></pre
>
</div>

<nav class="page bottom">
    <ul>
        <li class="current"><a href="index.html">Test</a></li>
        
        
        
    </ul>
</nav>

</body>
</html>